package.json

```diff
@@ -42,6 +42,14 @@
                     ],
                     "default": "./",
                     "description": "Specifies a relative path to a folder with node_modules which contains the TestCafe package"
+                },
+                "testcafeTestRunner.useLiveRUnner": {
```

Comment:
@romanresh I just noticed this case-typo. `useLiveRUnner` should be `useLiveRunner`. Do you want another PR, or do you prefer to handle minor changes like this another way?

Note: Fortunately, this doesn't change the functionality. The only side effect of the typo is that the settings.json will put a green squiggly under `testcafeTestRunner.useLiveRunner` if a lower-case "u" is used. Otherwise, if `"testcafeTestRunner.useLiveRunner": true` is set, the extension launches testcafe-live.

Reply:
Yep, I also overlooked the typo. Many thanks for your notice. Fixed.